config/mailinglist/rules/jdk.json

```diff
@@ -152,83 +136,23 @@
-            "src/java.base/macosx/classes/jdk",
-            "src/java.base/macosx/classes/sun/util",
-            "src/java.base/macosx/native/libjava/HostLocaleProviderAdapter_md.c",
-            "src/java.base/macosx/native/libjava/java_props_macosx.c",
-            "src/java.base/macosx/native/libjava/java_props_macosx.h",
-            "src/java.base/macosx/native/libjli",
-            "src/java.base/share/classes/module-info.java",
-            "src/java.base/share/classes/java/io/",
-            "src/java.base/share/classes/java/lang/",
-            "src/java.base/share/classes/java/math/",
-            "src/java.base/share/classes/java/text/",
-            "src/java.base/share/classes/java/time/",
-            "src/java.base/share/classes/java/util/",
-            "src/java.base/share/classes/jdk/internal/(?!icu)",
-            "src/java.base/share/classes/sun/invoke/util/",
-            "src/java.base/share/classes/sun/launcher/",
-            "src/java.base/share/classes/sun/reflect/",
-            "src/java.base/share/classes/sun/text/",
-            "src/java.base/share/classes/sun/util/BuddhistCalendar.java",
-            "src/java.base/share/classes/sun/util/PropertyResourceBundleCharset.java",
-            "src/java.base/share/classes/sun/util/calendar/",
-            "src/java.base/share/classes/sun/util/cldr",
-            "src/java.base/share/classes/sun/util/locale/",
-            "src/java.base/share/classes/sun/util/resources/",
-            "src/java.base/share/legal/",
-            "src/java.base/share/lib",
-            "src/java.base/share/native/include/classfile_constants.h.template",
-            "src/java.base/share/native/include/jni.h",
-            "src/java.base/share/native/launcher/main.c",
-            "src/java.base/share/native/libfdlibm/",
-            "src/java.base/share/native/libjava/",
-            "src/java.base/share/native/libjli/",
-            "src/java.base/share/native/libverify/",
-            "src/java.base/share/native/libzip/",
-            "src/java.base/unix/classes/java/io/",
-            "src/java.base/unix/classes/java/lang/",
-            "src/java.base/unix/classes/jdk/internal/loader/",
-            "src/java.base/unix/native/include",
-            "src/java.base/unix/native/jspawnhelper",
-            "src/java.base/unix/native/libjava/",
-            "src/java.base/unix/native/libjli/",
-            "src/java.base/unix/native/libjsig",
-            "src/java.base/windows/classes/java/io/",
-            "src/java.base/windows/classes/java/lang/",
-            "src/java.base/windows/classes/jdk/internal/loader/",
-            "src/java.base/windows/classes/sun/util",
-            "src/java.base/windows/native/common",
-            "src/java.base/windows/native/include",
-            "src/java.base/windows/native/libjava/",
-            "src/java.base/windows/native/libjimage",
-            "src/java.base/windows/native/libjli/",
+            "src/java.base/",
```

Comment:
This looks like core-libs-dev will be mailed for all changes in src/java.base, I don't think we want that as there it a lot of code in java.base that is maintained on other mailing lists.

Reply:
You are correct. I misread the statement in the bug report saying: `core-libs-dev cares about everything in src/java.base/*/native/libjava, no need to list specific files` as `core-libs-dev cares about everything in src/java.base/*, no need to list specific files`. My bad; will fix.